app/views/sessions/new.html.slim

```diff
@@ -0,0 +1,42 @@
+coffee:
+	root = exports ? this
+	root.signinCallback = (authResult) ->
+		if authResult['hd'] != "originate.com"
+			console.log "not originate"
+			disconnectUser authResult['access_token']
+		else 
+			if authResult['status']['signed_in']
+				# get user info
+				gapi.client.load 'plus', 'v1', () ->
+					request = gapi.client.plus.people.get {'userId':'me'}
+					request.execute (resp) ->
+						$.ajax
+							url: 		'users'
+							type:		'POST'
+							data:		resp
+							success:	(data, textStatus, jqHXR) ->
+											document.location = '/';
+							error:		(jqXHR, textStatus, errorThrown) ->
+											alert("Error: "+ errorThrown)
+			else
+				console.log "Error: Failed login"
```

Comment:
move this to a controller and instantiate it globally.  see `Vivify.HasManyAttachmentController` in the Vivify codebase for an example of this.